### DISCUSSION TITLE
Why would you not upload link prediction + hetero tutorial in the documentation?

### DISCUSSION THREAD

**Discussion by Jeriousman** [pyg-team/pytorch_geometric#3971] [General]
I think the link pred + hetero must be the one that most people are curious about. Do you have a plan to upload one?

**Comment by rusty1s** [comment]
Do you mean to explain [our example](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/hetero/hetero_link_pred.py) in more detail as part of the documentation? I think we could definitely consider integrating more information to our [this tutorial](https://pytorch-geometric.readthedocs.io/en/latest/notes/load_csv.html). Let me know if you are interested in working on integrating this with me.

**Reply by Jeriousman** [reply]
It was just a friendly suggestion. I dont think I am at the point where I can be so sure about what I am doing. But in the future, I would like to take a part for sure. Thank you!

**Reply by rusty1s** [reply]
No worries, and thanks for the suggestion. I labeled this feature request accordingly. Will try to integrate it.
